components/SEO.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
This allows us to add custom SEO tags to the files. Just need to pass strings for title/description/image in most cases - should simplify things a bit and again, provides consistency across pages